homeassistant/components/tibber/sensor.py

```diff
@@ -18,6 +18,7 @@
 ICON_RT = "mdi:power-plug"
 SCAN_INTERVAL = timedelta(minutes=1)
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+PARALLEL_UPDATES = 0
```

Comment:
0 means unlimited parallel updates. Is that what we want?

Reply:
It is one sensor per home, so it would never be that many.
Could maybe change it to 5?

Reply:
If it's only one sensor we probably can set it to unlimited.